samples/buck/sample.yaml

```
@@ -0,0 +1,65 @@
+sample:
+  description: Sample for nPM1300 PMIC BUCKs
+  name: nPM1300 PMIC BUCKs
+
+common:
+  integration_platforms:
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf52840dk_nrf52840
+  platform_allow: nrf5340dk_nrf5340_cpuapp nrf52840dk_nrf52840
+  tags: pmic
+  timeout: 30
+
+tests:
+  sample.pmic.buck.set_buck_voltage_eng_b:
```

```suggestion
  sample.buck.set_buck_voltage_eng_b:
```
Same in all instances below.

---

Fixed